docs/security-privacy-questionnaire.md

```diff
@@ -0,0 +1,105 @@
+
+The privacy implications of this are discussed [in the explainer](./docs/explainer.md#privacy-considerations).
+
+> 02.  Do features in your specification expose the minimum amount of information
+>      necessary to implement the intended functionality?
+
+Yes it is. The only information exposed is the payment link URI itself.
+
+The URI should contain the essential details required for the browser and payment clients to:
+- Identify the payment method: The URI scheme indicates the specific payment system being used (e.g., UPI, Bitcoin).
+- Initiate the payment: The URI may include parameters like payee address, amount, and currency, which are necessary to start the transaction.   
+- Validate the payment method: The browser uses the URI to check if the payment method is supported before prompting the user.
+
+> 03.  Do the features in your specification expose personal information,
+>      personally-identifiable information (PII), or information derived from
+>      either?
+
+No. Although the payment link URI may include parameters like payee address, amount, and currency, none of them belongs to above categories.
+
+> 04.  How do the features in your specification deal with sensitive information?
+
+The feature does not deal with sensitive information.
+
+> 05.  Do the features in your specification introduce state
+>      that persists across browsing sessions?
+
+No. The feature operates in a stateless manner.
+
+> 06.  Do the features in your specification expose information about the
+>      underlying platform to origins?
+
+No.
+
+> 07.  Does this specification allow an origin to send data to the underlying
+>      platform?
+
+Yes. The specification allows origins to send data to the platform only within the well-defined context of payment initiation, with user consent and security measures in place.
+
+> 08.  Do features in this specification enable access to device sensors?
+
+No.
+
+> 09.  Do features in this specification enable new script execution/loading
+>      mechanisms?
+
+No.
+
+> 10.  Do features in this specification allow an origin to access other devices?
+
+No.
+
+> 11.  Do features in this specification allow an origin some measure of control over
+>      a user agent's native UI?
+
+Possibly. When a valid payment link is detected, the browser may present a native UI to the user, allowing them to choose their preferred payment method or confirm the transaction. In this sense, the origin indirectly triggers the display of this UI by providing the payment link.
+
+> 12.  What temporary identifiers do the features in this specification create or
+>      expose to the web?
+
+None.
+
+> 13.  How does this specification distinguish between behavior in first-party and
+>      third-party contexts?
+
+It does not currently distinguish. The security measures (HTTPS, validation, etc.) apply regardless of the context, ensuring that payment links are handled securely in both cases.
```

Comment:
[bit.ly/html-payment-link-dd](https://bit.ly/html-payment-link-dd) says:

Security
-----
The payment link needs to be on on a page with the following properties:
 - ...
 - Only on top-level context (not inside of an iframe).

Reply:
Modified. Thanks!